src/dynarec/rv64/rv64_emitter.h

```diff
@@ -254,10 +261,13 @@ f28–31  ft8–11  FP temporaries                  Caller
 // 4-bytes[rs1+imm12] = rs2
 #define SW(rs2, rs1, imm12)         EMIT(S_type(imm12, rs2, rs1, 0b010, 0b0100011))
 
-#define PUSH1(reg)                  do {SD(reg, xRSP, -8); SUBI(xRSP, xRSP, 8);} while(0)
-#define POP1(reg)                   do {LD(reg, xRSP, 0); ADDI(xRSP, xRSP, 8);} while(0)
-#define PUSH1_32(reg)               do {SW(reg, xRSP, -4); SUBI(xRSP, xRSP, 4);} while(0)
-#define POP1_32(reg)                do {LWU(reg, xRSP, 0); ADDI(xRSP, xRSP, 4);} while(0)
+#define PUSH(reg)                  do {SD(reg, xRSP, -8); SUBI(xRSP, xRSP, 8);} while(0)
```

Comment:
I think you should have keeped the `PUSH1`/`POP1` for 2 reason:
1. to not confuse with a "native PUSH"
2. because we might be able to introduce an actual `PUSH2`/`POP2` using compressed opcodes

Reply:
I reverted this change.